artifact/gem5art/artifact/artifact.py

```diff
@@ -133,6 +133,8 @@ def registerArtifact(cls,
         when it was added to the database
         """
 
+        _db = getDBConnection()
```

Comment:
Let's change this name to `db` instead of `_db` now that it's not a private global variable

Reply:
done